### DISCUSSION TITLE
Trouble linking github repo

### DISCUSSION THREAD

**Discussion by johnhaslehurst4** [community/community#40108] [New to GitHub]
I am trying to link my github repo to a new project but it keeps giving me an error message saying its taking too long with a picture of a unicorn... (dont know if the unicorn is relevant). Has anyone else had this issue?

**Comment by DSDmark** [comment]
Hi @johnhaslehurst4,

Check out this discussion:-(https://github.com/community/community/discussions/38405#discussion-4542369)
